.github/workflows/non-cot-pr-build-and-e2e-tests.yml

```diff
@@ -80,11 +79,9 @@ jobs:
             - name: Setup WooCommerce Monorepo
               uses: ./.github/actions/setup-woocommerce-monorepo
 
-            - name: Load docker images and start containers with COT enabled.
+            - name: Load docker images and start containers with COT disabled.
```

Comment:
```
name: Load docker images and start containers
```

This job doesn't need to reference COT or HPOS now, so we can safely exclude any HPOS detail here, as we did in the e2e tests above.